xarray/coding/strings.py

```diff
@@ -130,19 +130,19 @@ def bytes_to_char(arr):
     if arr.dtype.kind != "S":
         raise ValueError("argument must have a fixed-width bytes dtype")
 
-    if is_duck_dask_array(arr):
-        import dask.array as da
-
-        return da.map_blocks(
-            _numpy_bytes_to_char,
-            arr,
-            dtype="S1",
-            chunks=arr.chunks + ((arr.dtype.itemsize,)),
-            new_axis=[arr.ndim],
-        )
-    else:
+    if not is_duck_dask_array(arr):
```

Comment:
+1 I liked the original a little better, because it makes it more obvious that we only import dask in the block where the argument is a dask array already.